doc/lua-filters.md

```diff
@@ -2403,6 +2403,59 @@ Usage:
 [TableHead]: #type-tablehead
 [Version]: #type-version
 
+## Chunk {#type-chunk}
+
+Part of a document, intended to be written into a separate
+file.
+
+Fields:
+
+`heading`
+:   heading text ([Inlines][])
+
+`id`
+:   identifier (string)
+
+`number`
+:   section number (integer)
+
+`path`
+:   target filepath for this chunk (string)
+
+`up`
+:   link to the enclosing section, if any (string|nil)
+
+`prev`
+:   link to the previous section, if any (string|nil)
+
+`next`
+:   link to the next section, if any (string|nil)
```

Comment:
Are these comments still accurate (also see the module below)?  `chunkNext` is now a `Maybe Chunk`, not just a link.

Reply:
Oh, it looks like you changed this in the last commit.
I wonder if it would be worth integrating those changes into this commit, so that the change isn't spread between two commits that aren't obviously related?

Reply:
I must have made a mistake in one of my rebases, these were not intended to be separate. I'll fix it.

Reply:
Done